lib/components/victory-area.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Dimensions, Platform } from "react-native";
+import { VictoryLabel, VictoryContainer, Area, VictoryClipContainer } from "victory-core-native";
```

Comment:
I've tried to run this branch, `unable to resolve module 'victory-core-native'`

is there anything I can do to help with this PR?